pkg/cloud/services/gc/cleanup_test.go

```diff
@@ -0,0 +1,654 @@
+							Values: []*string{aws.String("owned")},
+						},
+					},
+				}).DoAndReturn(func(awsCtx context.Context, input *rgapi.GetResourcesInput, opts ...request.Option) (*rgapi.GetResourcesOutput, error) {
+					return &rgapi.GetResourcesOutput{
+						ResourceTagMappingList: []*rgapi.ResourceTagMapping{
+							{
+								ResourceARN: aws.String("arn:aws:elasticloadbalancing:eu-west-2:1234567890:loadbalancer/app/aec24434cd2ce4630bd14a955413ee37"),
+								Tags: []*rgapi.Tag{
+									{
+										Key:   aws.String("kubernetes.io/cluster/eks-test-cluster"),
+										Value: aws.String("owned"),
+									},
+									{
+										Key:   aws.String(serviceNameTag),
+										Value: aws.String("default/svc1"),
+									},
+								},
+							},
+						},
+					}, nil
+				})
+			},
+			elbMocks: func(m *mocks.MockELBAPIMockRecorder) {},
+			elbv2Mocks: func(m *mocks.MockELBV2APIMockRecorder) {
+				m.DeleteLoadBalancerWithContext(gomock.Any(), &elbv2.DeleteLoadBalancerInput{
+					LoadBalancerArn: aws.String("arn:aws:elasticloadbalancing:eu-west-2:1234567890:loadbalancer/app/aec24434cd2ce4630bd14a955413ee37"),
+				}).Return(&elbv2.DeleteLoadBalancerOutput{}, nil)
+			},
+			ec2Mocks:  func(m *mocks.MockEC2APIMockRecorder) {},
+			expectErr: false,
+		},
+		{
+			name:         "ec2 cluster with ALB Service load balancer",
+			clusterScope: createUnManageScope(t),
+			rgAPIMocks: func(m *mocks.MockResourceGroupsTaggingAPIAPIMockRecorder) {
+				m.GetResourcesWithContext(gomock.Any(), &rgapi.GetResourcesInput{
+					TagFilters: []*rgapi.TagFilter{
+						{
+							Key:    aws.String("kubernetes.io/cluster/cluster1"),
+							Values: []*string{aws.String("owned")},
+						},
+					},
+				}).DoAndReturn(func(awsCtx context.Context, input *rgapi.GetResourcesInput, opts ...request.Option) (*rgapi.GetResourcesOutput, error) {
+					return &rgapi.GetResourcesOutput{
+						ResourceTagMappingList: []*rgapi.ResourceTagMapping{
+							{
+								ResourceARN: aws.String("arn:aws:elasticloadbalancing:eu-west-2:1234567890:loadbalancer/app/aec24434cd2ce4630bd14a955413ee37"),
+								Tags: []*rgapi.Tag{
+									{
+										Key:   aws.String("kubernetes.io/cluster/cluster1"),
+										Value: aws.String("owned"),
+									},
+									{
+										Key:   aws.String(serviceNameTag),
+										Value: aws.String("default/svc1"),
+									},
+								},
+							},
+						},
+					}, nil
+				})
+			},
+			elbMocks: func(m *mocks.MockELBAPIMockRecorder) {},
+			elbv2Mocks: func(m *mocks.MockELBV2APIMockRecorder) {
+				m.DeleteLoadBalancerWithContext(gomock.Any(), &elbv2.DeleteLoadBalancerInput{
+					LoadBalancerArn: aws.String("arn:aws:elasticloadbalancing:eu-west-2:1234567890:loadbalancer/app/aec24434cd2ce4630bd14a955413ee37"),
+				}).Return(&elbv2.DeleteLoadBalancerOutput{}, nil)
+			},
+			ec2Mocks:  func(m *mocks.MockEC2APIMockRecorder) {},
+			expectErr: false,
+		},
+		{
+			name:         "eks cluster full test",
```

Comment:
Could you find a more descriptive name for this test?

Reply:
Changed the name